youtube_dl/utils.py

```diff
@@ -1596,6 +1600,15 @@ def month_by_name(name):
         return None
 
 
+def month_by_french_name(name):
+    """ Return the number of a month by French name """
+
+    try:
+        return FRENCH_MONTH_NAMES.index(name) + 1
+    except ValueError:
+        return None
```

Comment:
This should be generalized in the first place. Now it simply a code duplication.